index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+
+    <title> PORTFOLIO</title>
+</head>
+
+<body>
+
+    <!-- NAVBAR -->
+    <div class="container">
+        <header class="header">
+            <a href="#"><i class="fa-solid fa-m color"></i></a>
+            <nav>
+                <button type="button"><img src="images/Menu.png" alt=""></button>
+            </nav>
+        </header>
+
+        <!-- INTRODUCTION -->
+        <section class="intro-text">
+            <h1 class="text"> I’m Muskan</h1>
+            <h2 class="text">Glad to see you!</h2>
+            <p class="para">I’m a software developer! I can help you build a product , feature or website Look through
+                some
+                of my work and experience!
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
+
+        <!-- SKILL -->
+        <div class="links ">
+            <p>LET’S CONNECT</p>
+            <div class="connect">
+                <ul class="socialmedia">
+                    <li><a href="#" target="_blank"> <i class="fa-brands fa-twitter icon"></i></a> </li>
+                    <li><a href="#" target="_blank"> <i class="fa-brands fa-github icon"></i></a></li>
+                    <li><a href="#" target="_blank"><i class="fa-brands fa-linkedin-in icon"></i></a></li>
+                    <li><a href="#" target="_blank"></a></li>
+                </ul>
+            </div>
+        </div>
+    </div>
+
+    <div class="grid-container">
+        <!-- Card -->
+        <div class="cards">
```

Comment:
- [ **OPTIONAL** ] I think it would be better not to use the fixed-width on all your sections styling to make your page responsive and this will make the work easy when you will start the **Desktop** version.